server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/decent/sources/coinmarketcap/ICoinmarketcapRateAPI.java

```diff
@@ -0,0 +1,15 @@
+package com.generalbytes.batm.server.extensions.extra.decent.sources.coinmarketcap;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/v1")
+@Produces(MediaType.APPLICATION_JSON)
+public interface ICoinmarketcapRateAPI {
+    @GET
+    @Path("/ticker/{cryptocurrency}/?convert={fiatcurrency}")
+    CoinmarketcapResponse[] getRequest(@PathParam("cryptocurrency") String cryptoCurrency, @PathParam("fiatcurrency") String fiatCurrency);
+}
```

Comment:
Rather than duplicating code, add DCT support to the original CoinmarketcapRateSource.